Dockerfile

```diff
@@ -84,7 +84,9 @@ RUN wget ${archive_url} && \
     chown -R www-data:root /var/www/html && \
     rm -r ${cachet_ver}.tar.gz && \
     php /bin/composer.phar global require "hirak/prestissimo:^0.3" && \
-    php /bin/composer.phar install -o && \
+    php /bin/composer.phar install --no-dev -o && \
+    php /bin/composer.phar update barryvdh/laravel-debugbar && \
+    php /bin/composer.phar dump-autoload && \
```

Comment:
@camilb just curious what these two composer lines do here? I am by no means an expert at composer  / laravel practices, so this is mostly just for my own curiosity.

Reply:
@djdefi sorry, some leftovers from first attempt to make it work with v2.4. Pushed a new commit